web_calendar_custom/static/src/js/web_calendar_custom.js

```diff
@@ -0,0 +1,114 @@
+/*
+    © 2018 Savoir-faire Linux <https://savoirfairelinux.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+
+
+odoo.define('web_calendar_custom.renderer_setting', function (require) {
+"use strict";
+
+    var rpc = require('web.rpc');
+    var AbstractRenderer = require('web.AbstractRenderer');
+    var core = require('web.core');
+
+    var _t = core._t;
+
+
+    String.prototype.toHHMMSS = function () {
+        var hour_num = parseFloat(this); // don't forget the second param
```

Comment:
@lgharib you still need this comment? // don't forget the second param

Reply:
No I just removed it